jekyll/_cci2/monitoring.adoc

```diff
@@ -105,6 +173,79 @@ image::metrics_datadog1.png[Enable DataDog]
+
+// Should this be "in addition to" or "instead of" ^^ This sentence doesn't make sense to me - After chatting withe George at the docs meeting I think customers can either get all metrics we provide to Cloudwatch or Datadog - OR they can enable Custom metrics and edit the configuration file to ONLY add the metrcis they want to see... ?
+
+.Custom Metrics
+image::metrics-custom.png[Custom Metrics]
+
+=== Customizing Metrics
+
+Following are the steps required to customize which of these metrics you wish to receive:
+
+. Check to enable Use Custom Telegraf Metrics from the Management Console settings
++
+.Custom Metrics
+image::metrics-custom.png[Custom Metrics]
+. SSH into the Services machine
+. Add the following to `/etc/circleconfig/telegraf/statsd.conf`
++
+```
+[[inputs.statsd]]
+        service_address = ":8125"
+        parse_data_dog_tags = true
+        metric_separator = "."
+        namepass = []
+```
+. Under `namepass` add any metrics you wish to receive, for example, if you are only interested in metics for VM service, your cofnfig should resemble:
++
+```
+[[inputs.statsd]]
+        service_address = ":8125"
+        parse_data_dog_tags = true
+        metric_separator = "."
+        namepass = [
+          "circle.vm-service.vm.assigned-vm",
+            "circle.vm-service.vms.delete.status",
+            "circle.vm-service.vms.get.status",
+            "circle.vm-service.vms.post.status"
+          ]
+```
+. Restart the telegraf container by running: `sudo docker restart telegraf`
+. Navigate to your Management Console settings page, scroll down to save and restart your installation.
+
+=== Configuring Custom Metrics
+
+Configuration options are based on Telegraf's documented output plugins. See their documentation https://github.com/influxdata/telegraf/tree/release-1.10#output-plugins[here]. For example, if you would like to use the InfluxDB Output Plugin you would need to follow these steps:
+
+. SSH into the Servics Machine
+. cd `/etc/circleconfig/telegraf/influxdb.conf`
+. Adding the desired outputs, for example:
++
+```yaml
+[[output.influxdb]]
+  url = "http://52.67.66.155:8086"
+  database = "testdb"
+```
+. Run `docker restart telegraf` to restart the container to load or reload any changes.
+
+You may check the logs by running `docker logs -f telegraf` to confirm your output provider (e.g. influx) is listed in the configured outputs. Additionally, if you would like to ensure that all metrics in an installation are tagged against an environment you could place the following code in your config:
+
+```yaml
+[global_tags]
+Env="<staging-circleci>"
+```
+
+Please see the InfluxDB https://github.com/influxdata/influxdb#installation[documentation] for default and advanced installation steps.
+
+NOTE: Any changes to the config will require a restart of the system.
+
+//what system? the whole CircleCI system? incurring downtime?
```

Comment:
Yup, the whole CircleCI system. It would incur downtime.

Reply:
Done